src/agent_toolkit/forestadmin/agent_toolkit/resources/collections/filter.py

```diff
@@ -223,9 +222,7 @@ def _parse_value(jsoned_filters, collection):
         new_value = literal_eval(str(jsoned_filters["value"]))
 
     elif schema["column_type"] == PrimitiveType.BOOLEAN:
-        new_value = (
-            strtobool(jsoned_filters["value"]) if isinstance(jsoned_filters["value"], str) else jsoned_filters["value"]
```

Comment:
no side-effects on older versions ?

Reply:
That depend on what the front-end sends, not on the version. After debugging, strtobool was never called because it's always a boolean type.